src/services/map-events-mananger/consts/pickOptions.enum.ts

```diff
@@ -3,10 +3,12 @@
  *  PICK_FIRST  - first entity will be picked . use Cesium.scene.pick()
  *  PICK_ONE    - in case a few entities are picked plonter is resolved . use Cesium.scene.drillPick()
  *  PICK_ALL    - all entities are picked. use Cesium.scene.drillPick()
+ *  MULTI_PICK  - every selected entity is picked. If chosen twice will be unselected. use Cesium.scene.pick()
```

Comment:
Consider creating a more general MapSelectionService which will handle all kinds of selections. This service will handle all selected map entities and will have functionality of:
-multi selection (over a specified area on the map for example - supported in cesium)
-standard one entity selection
-add to selected entities using CTRL for example
-deselect specific entity \ entities \ all entities
@eitanfr  what do you think?